### DISCUSSION TITLE
Is it possible to 'duplicate' a layer so that it covers the whole map.

### DISCUSSION THREAD

**Discussion by NadineSarah** [visgl/deck.gl#6214] [Q&A] [answered]
Hi, 
Due to the width of my div, the base map may be repeated (see image). Is it possible to achieve the same behavior with a deck gl layer (so that it covers the whole base map)?
![image](https://user-images.githubusercontent.com/39406788/134025915-2d62b4d0-5165-4040-a12a-7b0e79e25178.png)

**Comment by Pessimistress** [comment, marked as answer]
https://deck.gl/docs/api-reference/core/map-view#repeat
